### PR TITLE
Fix for AWAY raws displaying empty output in channel window

### PR DESCRIPTION
**Pull request by haxxordan**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Hi

There's a few things that need to be addressed:
* Changes made to script files are to be made to the main script file as well as the translation script file. (*mrc and *ptm) - You are only patching to .ptm. As there's no contributing.md file in our repo, we're a bit unclear as to what we expect from PR's. So this one is on us.
* There's no corresponding issue logged where we can see/discuss the error at hand. What exactly is happening? Once again, we haven't specified a contributing.md file where we describe such things. But if you can help us out here and provide some more background information about what is happening, that would be much appreciated!
* I am confused about $numeric == 0. Imho, there's no $numeric 0. Maybe you can explain what you see in an issue? See above.
* Parentheses are important. Ideally I would write it as `if (($numeric !isnum) || ($numeric == 0)) return`

best regards!

tldr;
Could you help clarify $numeric 0 and maybe provide some extra information about what is going wrong? Screenshot would be helpful and or steps to reproduce the error!

**Comment by ghost** [comment]
Ok after asking for some more information it seems that there are raw events where the numeric is alphabetic non-numeric, so is given 0...for example:

`raw account:*:{}` where someone's address changes due to enabling their cloak...
